kornia/nerf/rays.py

```diff
@@ -19,12 +19,12 @@ class RaySampler:
         ndc: convert ray parameters to normalized device coordinates: bool
         device: device for ray tensors: Union[str, torch.device]
     """
-    _origins: Optional[Tensor] = None  # Ray origins in world coordinates (*, 3)
-    _directions: Optional[Tensor] = None  # Ray directions in world coordinates (*, 3)
-    _directions_cam: Optional[Tensor] = None  # Ray directions in camera coordinates (*, 3)
-    _origins_cam: Optional[Tensor] = None  # Ray origins in camera coordinates (*, 3)
-    _camera_ids: Optional[Tensor] = None  # Ray camera ID
-    _points_2d: Optional[Tensor] = None  # Ray intersection with image plane in camera coordinates
+    _origins: Tensor  # Ray origins in world coordinates (*, 3)
+    _directions: Tensor  # Ray directions in world coordinates (*, 3)
+    _directions_cam: Tensor  # Ray directions in camera coordinates (*, 3)
+    _origins_cam: Tensor  # Ray origins in camera coordinates (*, 3)
+    _camera_ids: Tensor  # Ray camera ID
+    _points_2d: Tensor  # Ray intersection with image plane in camera coordinates
```

Comment:
i believe this should be initialized

Reply:
The tests did not complain, maybe will need to correct the tests as well?